release/models/system/openconfig-system-controlplane.yang

```diff
@@ -0,0 +1,141 @@
+       is sufficient, the CPU-facing interface uses the specified scheduler
+       to determine how to drain queues. If more granular remapping is required
+       (e.g., to classify control-plane traffic more granularly), a user specifies
+       an alternate classifier that is used to reclassify traffic into
+       a new set of forwarding-groups (and hence queues) that can subsequently
+       be scheduled by the specified scheduler.
+       The specified control-plane ACL is applied to traffic received by the
+       control-plane of the system.";
+
+    oc-ext:openconfig-version "0.1.0";
+    oc-ext:catalog-organization "openconfig";
+    oc-ext:origin "openconfig";
+
+    revision "2021-08-02" {
+        description
+          "Initial revision.";
+        reference
+          "0.1.0";
+    }
+
+    grouping system-controlplane-top {
+        description
+          "Top-level structural grouping for control-plane traffic policies.";
+
+        container control-plane-traffic {
+            description
+              "Policies and configuration relating to the traffic destined towards
+               the system control-plane.";
+
+            container config {
+                description
+                  "Configuration parameters relating to control-plane traffic.";
+
+                uses system-controlplane-config;
+            }
+
+            container state {
+                config false;
+                description
+                  "Operational state parameters relating to control-plane traffic.";
+
+                uses system-controlplane-config;
+            }
+        }
+    }
+
+    grouping system-controlplane-config {
+        description
+          "Grouping for configuration parameters relating to the system
+           control-plane.";
+
+        leaf classifier-name {
+            type leafref {
+                path "/oc-qos:qos/oc-qos:classifiers/oc-qos:classifier/" +
+                "oc-qos:config/oc-qos:name";
+            }
+            description
+              "Reference to a classifier that is used to classify traffic
+               destined to the control-plane of the system.
+               This classifier determines how packets that match each terms
+               are classified into forwarding groups, and subsequently into
+               queues to be forwarded.";
+        }
+
+        leaf scheduler {
+            type leafref {
+                path "/oc-qos:qos/oc-qos:scheduler-policies/oc-qos:scheduler-policy/" +
+                "oc-qos:config/oc-qos:name";
+            }
+            description
+              "Reference to a scheduler policy that determines rate limits, or
+               shaping of packets towards the control-plane.";
+        }
+
+        leaf acl-v4 {
+            type leafref {
+                path "/oc-sets:defined-sets/oc-sets:ipv4-prefix-sets/" +
+                "oc-sets:ipv4-prefix-set/oc-sets:config/oc-sets:name";
```

Comment:
Should this be the leafref of an ACL from the `acl-set`? How does a prefix-set associate itself with an ACL here?

Reply:
Agreed. This should reference `/acl/acl-sets/acl-set/name`.